### PR TITLE
[ez-ci] Add behat testing of Platform UI

### PR DESCRIPTION
**Pull request by andrerom**

### CODE REVIEW

File: bundle.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+tmp
```
Comment (crevillo) [inline bundle.txt]: curiosity: what is this for?

Comment (andrerom) [inline bundle.txt]: Tmp file, won't be there in the future. Just to enable testing atm.

Comment (crevillo) [inline bundle.txt]: ok. thanks.

File: Features/Standard/platformui.feature [hunk 2]
```diff
@@ -1,25 +1,25 @@
 Feature: Basic PlatfomrUi interaction tests
 
-    @javascript
+    @common @javascript
     Scenario: As a admin User, I want to login to PlatformUI
         Given I go to PlatformUI app with username "admin" and password "publish"
         Then I should see "Welcome to eZ Platform" title
 
-    @javascript
+    @commom @javascript
```
Comment (vidarl) [inline Features/Standard/platformui.feature]: Typo : commo**m**

This typo is 2 duped two more times in the PR

Comment (andrerom) [inline Features/Standard/platformui.feature]: ok, fixing

**Pull request merged**